packages/components/src/components/input-field/input-field.jsx

```diff
@@ -45,6 +45,7 @@ const InputField = ({
     min_value,
     name,
     format,
+    onBlurHandler,
```

Comment:
This can just be `onBlur`, like we have in other components.

Reply:
Will rename it in this file.